fineract-core/src/main/java/org/apache/fineract/infrastructure/core/data/DataValidatorBuilder.java

```diff
@@ -1056,4 +1056,12 @@ public void throwValidationErrors() throws PlatformApiDataValidationException {
             throw new PlatformApiDataValidationException(dataValidationErrors);
         }
     }
+
+    public static ApiParameterError buildValidationParameterApiError(final String resource, final String parameterName,
```

Comment:
I dont think we need this. You can just simply force return and the `baseDataValidator` would be processed till that point...

Reply:
I've moved almost all the cases, just one to use this new method